### ISSUE TITLE
dns/ddclient: no-ip broken

### ISSUE THREAD

**Issue by PDy14n** [opnsense/plugins#2806]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [X] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [X] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [X] The title contains the plugin to which this issue belongs

**Describe the bug**
My no-ip configuration with the new ddclient gives errors, while the dyndns config works fine. It also doesnt show up in the dashboard with the widget and/or service.

**To Reproduce**

- have a valid dyndns no-ip configuration
- uninstall dyndns plugin
- install ddclient plugin
- configure ddclient settings with: 
1. Account - same as in the dyndns config
2. general settings - interval: 300
check ip method: no-ipv4 (dyndns doesnt seem to work for me either, but doesnt give the` [meta sequenceId="1"] WARNING: cannot connect to ip1.dynupdate.no-ip.com:80 socket: IO::Socket::INET: Bad hostname 'ip1.dynupdate.no-ip.com'   `)

**Expected behavior**
The same as with dyndns: an update on my no-ip cached ip, or at least log indications that the action is not necessary like my first log-line.

**Relevant log files**
Logs give me the following (did some reinstalls of the plugin, and change of settings, as well as reboots... but to no avail):
```
2022-02-03T21:36:57	Notice	ddclient[79495]	 24746 - [meta sequenceId="1"] WARNING:  file /usr/local/etc/ddclient.conf: file /usr/local/etc/ddclient.conf must be accessible only by its owner (fixed).
2022-02-03T21:16:56	Notice	ddclient[79495]	 49526 - [meta sequenceId="1"] WARNING:  file /usr/local/etc/ddclient.conf: file /usr/local/etc/ddclient.conf must be accessible only by its owner (fixed).
2022-02-03T21:01:56	Notice	ddclient[79495]	 89856 - [meta sequenceId="2"] WARNING:  found neither ipv4 nor ipv6 address
2022-02-03T21:01:56	Notice	ddclient[79495]	 86145 - [meta sequenceId="1"] WARNING:  cannot connect to ip1.dynupdate.no-ip.com:80 socket: IO::Socket::INET: Bad hostname 'ip1.dynupdate.no-ip.com'
2022-02-03T20:55:21	Notice	ddclient[36769]	 82331 - [meta sequenceId="1"] WARNING:  file /usr/local/etc/ddclient.conf: file /usr/local/etc/ddclient.conf must be accessible only by its owner (fixed).
2022-02-03T20:50:21	Notice	ddclient[36769]	 48716 - [meta sequenceId="2"] WARNING:  found neither ipv4 nor ipv6 address
2022-02-03T20:50:21	Notice	ddclient[36769]	 44592 - [meta sequenceId="1"] WARNING:  cannot connect to ip1.dynupdate.no-ip.com:80 socket: IO::Socket::INET: Bad hostname 'ip1.dynupdate.no-ip.com'
2022-02-03T20:46:31	Notice	ddclient[70934]	 35353 - [meta sequenceId="1"] WARNING:  file /usr/local/etc/ddclient.conf: file /usr/local/etc/ddclient.conf must be accessible only by its owner (fixed).
2022-02-03T20:37:56	Notice	ddclient[54346]	 19 - [meta sequenceId="1"] WARNING:  file /usr/local/etc/ddclient.conf: file /usr/local/etc/ddclient.conf must be accessible only by its owner (fixed).
2022-02-03T20:32:56	Notice	ddclient[54346]	 57229 - [meta sequenceId="1"] WARNING:  updating ddnsact54858997974632315688744163132698956323.ddns.me: nochg: No update required; unnecessary attempts to change to the current address are considered abusive
```
**Additional context**
The things I did today: 
- install the new 22.1 release from 21.7.
- install the plugin ddclient
- uninstall the dyndns plugin
- configure the ddclient
- reinstall the ddclient (should fix the "not showing in dashboard" issue as written in https://github.com/opnsense/plugins/issues/2783#issuecomment-1025585476 , but for me this doesnt solve it)
- reboot
- change the check ip method
- revalidate my account credentials

**Environment**
OPNsense 22.1 (amd64, OpenSSL)
Intel(R) Core(TM) i3-10100 CPU @ 3.60GHz (4 cores, 8 threads)

**Comment by AdSchellevis** [comment]
could be a dns configuration issue on the machine, you can always check if `/usr/local/etc/ddclient.conf` contains the correct config, same service fetches an address without issues on my end.

**Comment by PDy14n** [comment]
Thank you for your quick response! 
After looking back at my settings I finally used my brain and turned on verbose logging for the Dynamic dns under the advanced mode slider menu. Turns out it was working fine all along, but wasn't showing any time updates on my no-ip account because the ip was already set to the current one. The previous dyndns plugin showed me the status on the dashboard widget, but now I only went with the info on no-ip and incorrectly assumed it wasn't working. 

So although this was entirely my fault, I think it would be nice to see the widget back in de dashboard like before so we can quickly glance over at the cached ip.
secondly, the service still isn't showing up under the services for me, but it's running fine, so that might just be a GUI thing.

Anyways, this turned out to be a non-issue, so it can be closed safely. Thanks for your time.
